src/styles/global.css

```
@@ -23,6 +23,9 @@ h1 {
 /* nav styles */
 
 .hamburger {
+  background: transparent;
+  display: block;
+  border: none;
   padding-right: 20px;

```

If we want to keep the same rendering as before, we also need to remove the button default padding:
```suggestion
  padding: 0;
```
I don't know why we have that `padding-right`. It is currently unused (I also tested with the current version). Maybe at some point both the hamburger and the theme switcher was meant to be inlined but this never happened? I tried to check the history and it seems it's there from the beginning but I don't see any obvious reason why (see https://github.com/withastro/docs/pull/1089).